piker/_daemon.py

```diff
@@ -87,6 +88,18 @@ async def open_pikerd(
             yield _services
 
 
+@asynccontextmanager
+async def maybe_open_runtime(
```

Comment:
Somewhat hacky and I wonder if we should offer this from `tractor` itself?

The trick is being able to ping the actor registry when you're code can run as a root out of process tree.